lib/python/rosie/ws_client_auth.py

```diff
@@ -246,8 +246,8 @@ class RosieWSClientAuthManager(object):
         "gnomekeyring": GnomekeyringStore,
         #KeyringStore,
     }
-    PROMPT_USERNAME = "Username for %(prefix)r: "
-    PROMPT_PASSWORD = "Password for %(username)s at %(prefix)r: "
+    PROMPT_USERNAME = "Username for %(host)r: "
```

Comment:
It would be better to retain the prefix:

"Username for %(prefix)r - %(root)r: "